packages/example-ui/src/components/common/Checkout/CardFormComponent.tsx

```diff
@@ -96,40 +85,34 @@ function CardFormComponent(props: { params: CreatePaymentParams }) {
   
   const classes = useStyles();
 
-  async function onSubmit(formValues: any) {
-    setShowPaymentMethodErrors(true);
-    if (submitting || paymentMethodErrors?.length) return;
+  const onSubmit = async (formValues: any) => {
+    if (submitting) return;
 
     setSubmitting(true);
 
     const cardForm = (cardFormRef as any).current;
     const paymentMethodMetadata = { ...formValues };
     const tokenizeResponse = await cardForm.tokenize(clientId, paymentMethodMetadata, params.sellerAccountId);
 
-    if (tokenizeResponse.token) {
+    if (tokenizeResponse.id) {
```

Comment:
I don't think this is right based on the types here: https://github.com/justifi-tech/web-component-library/blob/main/stencil-library/src/components/payment-method-form/payment-method-responses.ts#L21

Reply:
It is the same value.